sandbox/tyrianapp.js

```diff
@@ -0,0 +1,3 @@
+import {TyrianApp} from './.js/target/scala-3.1.1/sandbox-fastopt.js';
+
+TyrianApp.launch("myapp", {"message": "Hello, Tyrian!"});
```

Comment:
This is how we launch the app, tell it what container id to attach itself too, and pass in some flags.